R/praise.R

```diff
@@ -62,15 +6,21 @@
 #'
 #' @examples
 #' replicate(5, glue::glue("Random praise: {random_praise()}"))
-#' replicate(5, glue::glue("Random encouragement: {random_encourage()}"))
+#' replicate(5, glue::glue("Random encouragement: {random_encouragement()}"))
 #' @export
 #' @describeIn praise Returns a random praise message
 random_praise <- function() {
-  sample(.praise, 1)
+  sub("^Correct! ", "", learnr::random_praise())
 }
 
 #' @export
-#' @describeIn praise Returns a random encouragement message
+#' @describeIn praise Returns a random encouragement message, included for 
+#'   backwards compatibility. 
 random_encourage <- function() {
-  sample(.encourage, 1)
+  lifecycle::deprecate_soft("0.2.1", "random_encourage()", "random_encouragement()")
+  random_encouragement()
 }
+
+#' @export
+#' @describeIn praise Returns a random encouragement message
+random_encouragement <- learnr::random_encouragement
```

Comment:
Similar to https://github.com/rstudio/shiny/blob/2590cf3895632033534372357c1deee713152a62/R/reexports.R#L9-L11

```suggestion
#' @describeIn praise Returns a random encouragement message
#' @export
learnr::random_encouragement
```

Reply:
There is some weird wonky behavior that can come up when the `gradethis` package doesn't update and `learnr` changes its API.

The installed binary of `gradethis` will hold on to the `learnr` API when the binary was built, even though there might be a new `learnr` API since the original build of the `gradethis` binary.

To get around this, we can reexport an existing method using this `roxygen2` trick.